README.md

```diff
@@ -13,10 +13,10 @@ https://github.com/rfordatascience/tidytuesday/raw/master/data/2022/2022-01-18/c
 You may use the included `chocolate_data_download.R` to download, to use it:
 
 ```
-Rscript data/chocolate_data_download.R --url = https://github.com/rfordatascience/tidytuesday/raw/master/data/2022/2022-01-18/chocolate.csv --file_path = data/raw/chocolate.csv
+Rscript data/src/chocolate_data_download.R --url = https://github.com/rfordatascience/tidytuesday/raw/master/data/2022/2022-01-18/chocolate.csv --file_path = data/raw/chocolate.csv
```

Comment:
I think should be `src/chocolate_data_download.R` instead of `data/src/chocolate_data_download.R` now? :thinking:

Reply:
Yeah you are right.